modules/ROOT/pages/dw-dates-functions-datetime.adoc

```diff
@@ -41,7 +41,7 @@ import * from dw::core::Dates
 output application/json
 ---
 {
-    newDateTime: dateTime({year: 2012, month: 10, day: 11, hour: 12, minutes: 30, seconds: 40 , timeZone: |-03:00|})
+    newDateTime: dateTime({year: 2012, month: 10, day: 11, hour: 12, minutes: 30, seconds: 40 , timeZone: &#124;-03:00&#124;})
```

Comment:
TODO: CHECK if the HTML code gets transformed to a pipe in this example, prob not.

Reply:
fixed in source, master